llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -518,7 +525,7 @@ bool PostPipeliner::scheduleFirstIteration(PostPipelinerStrategy &Strategy) {
 
     scheduleNode(SU, Actual);
     Info[N].Scheduled = true;
-    LLVM_DEBUG(dbgs() << "Scoreboard\n"; Scoreboard.dumpFull(););
+    DEBUG_FULL(dbgs() << "Scoreboard\n"; Scoreboard.dumpFull(););
```

Comment:
Note: dumping the scoreboard is pretty expensive, it brings compile-time up from tens of milliseconds to seconds